### PR TITLE
Fixes #2757 - Add markdown-it to issues on production

### LINKED ISSUE #2757: [v15.4.0] [broken release] MarkdownSanitizerMixin
@miketaylr @marimeireles 

We are not out yet of the woods. :) #2727 is not completely over. :)
```
 ReferenceError: MarkdownSanitizerMixin is not defined[Learn More] issues.min.js:7:15902

    <anonymous> https://webcompat.com/js/dist/issues.min.js?3054c48ab8310c374912a235d2407e15:7 
```

### PR DESCRIPTION
**Pull request by marimeireles**
Mike found the answer! :)

r? @karlcow

### CODE REVIEW

**Comment by marimeireles** [comment]
Do I need to clean my commit history by rebasing?

**Comment by miketaylr** [comment]
> Do I need to clean my commit history by rebasing?

If you can, that would be nice (before we merge). :))

**Comment by miketaylr** [comment]
> r? @karlcow

(going to steal this review)

File: grunt-tasks/concat.js [hunk 1]
```diff
@@ -53,7 +57,6 @@ module.exports = function(grunt) {
         "<%= jsPath %>/lib/models/label-list.js",
         "<%= jsPath %>/lib/models/issue.js",
         "<%= jsPath %>/lib/mixins/pagination.js",
-        "<%= jsPath %>/lib/mixins/extend-md-sanitizer.js",
```
[the remark this answers is not in the document]
Comment (marimeireles) [reply grunt-tasks/concat.js]: ok :)

**Comment by karlcow** [comment]
❤️ to the two of you! Thanks @marimeireles and @miketaylr

**Pull request merged**